acorn/src/expression.js

```diff
@@ -964,7 +964,6 @@ pp.parseFunctionBody = function(node, isArrowFunction, isMethod, forInit) {
 
   if (isExpression) {
     node.body = this.parseMaybeAssign(forInit)
-    node.expression = true
```

Comment:
I'm sorry if I'm wrong, but I think the expression property is required for ArrowFunctionExpression.

https://github.com/estree/estree/blob/master/es2015.md#arrowfunctionexpression

Reply:
Thanks! Realized that after I open this, so I put this into draft. And opened an issue about it in ESTree https://github.com/estree/estree/issues/323

Reply:
You are right `ArrowFunctionExpression.expression` is required.